devito/passes/iet/definitions.py

```diff
@@ -219,6 +219,17 @@ def _alloc_object_array_on_low_lat_mem(self, site, obj, storage):
 
         storage.update(obj, site, allocs=decl)
 
+    def _alloc_special_type_on_low_lat_mem(self, site, obj, storage):
```

Comment:
That's still the exact same one as `_alloc_object_array_on_low_lat_mem` why does it needs a separate one